.asf.yaml

```diff
@@ -14,6 +14,9 @@ github:
     rebase: true
   autolink_jira:
     - MNG
+  del_branch_on_merge: true
+  features:
+    issues: true
```

Comment:
this activation is not ready for merge to me: https://lists.apache.org/thread/56mhgp2kynylo5ctpdk5m1qczjzj6wln

Reply:
Are we done now? @hboutemy @slachiewicz I think we should enable GH issues.